src/sass/components/_task-table.scss

```diff
@@ -0,0 +1,15 @@
+// Task Tables
+// ===========
+
+// add a tiny top margin to create visual appearance of centered text
+.project-task-status {
+  > div {
```

Comment:
The div shows an empty class attribute in dev tools, is there a way to pass it a class? This works fine but doesn't feel totally reliable. Even `div[title]` might be a bit more specific to the icon?

Reply:
There is not a way to pass that div a class.